cpu/lm4f120/include/cpu_conf.h

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2015 Rakendra Thapa <rakendrathapa@gmail.com
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup        cpu_lm4f120
+ * @ingroup         cpu
+ * @brief           CPU specific implementations for the Stellaris Launchpad LM4F120 board
+ * @{
+ *
+ * @file
+ * @brief           CPU specific hwtimer configuration options
+ *12
+ * @author          Rakendra Thapa <rakendrathapa@gmail.com>
+ */
+
+#ifndef __CPU_CONF_H
+#define __CPU_CONF_H
+
+#include "hw_ints.h"
+#include "hw_memmap.h"
+#include "hw_nvic.h"
+#include "hw_sysctl.h"
+#include "hw_types.h"
+#include "lm4f120h5qr.h"
```

Comment:
could you make this include dependable on the `CPU_MODEL`? This will make it easier in the future to include other LMxx CPUs...